src/main/webapp/player.css

```diff
@@ -0,0 +1,27 @@
+/** Player Sprite **/
+#image-container {
+  position: absolute;
+  height : 550px;
+  left : 70%;
+  top : 250px;
+  float : left;
+}
+
+#pixel-avatar {
+  position: absolute;
+  width : 260px;
+  left : 10px;
+  top : 160px;
+  height : 260px;
+  z-index: 0;
+}
+
+#player-picture {
+  object-fit: cover;
+  position: absolute;
+  border-radius: 50%;
+  left : 50px;
+  width : 170px;
+  height : 185px;
+  z-index: 1;
+}
```

Comment:
why do we need a separate css style sheet if these classes are similar to the ones in style.css

Reply:
Honestly just to make it a bit easier for me to look at, our style.css file is getting very long.

Reply:
Can we break our style.css out then?